app/views/idv/phone/new.html.slim

```diff
@@ -1,6 +1,6 @@
 - title t('idv.titles.phone')
 
-h1.h2.my0 = t('idv.titles.session.phone')
+h1.h3.my0 = t('idv.titles.session.phone')
```

Comment:
I'd rather see a class attached to the `h2` suggesting a smaller size that select the heading level based on the visual design. But I'm not sure how much we try to follow good semantics in our HTML.

Reply:
Yup. You're right. I got stuck on the class name looking like the element name.